src/views/forms/AccountManagementForms/SourceSelectMenu.tsx

```diff
@@ -51,7 +53,12 @@ export const SourceSelectMenu = ({
     (type === TransferType.deposit ? depositOptions : withdrawalOptions)?.exchanges?.toArray() ??
     EMPTY_ARR;
 
-  const cctpTokensByChainId = useMemo(() => getMapOfLowestFeeTokensByChainId(type), [type]);
+  const skipEnabled = useStatsigGateValue(StatSigFlags.ffSkipMigration);
+
+  const cctpTokensByChainId = useMemo(
```

Comment:
lowest fees is different than cctpTokens. if cctpDeposit/withdrawal only is set to true, it filters out ethereum